### PR TITLE
Fix New-GceFireWallProtocol -Port parameters

### PR DESCRIPTION
**Pull request by quoctruong**
Also fix some Logging tests. Remove some containers tests (they take a long time to run).

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-powershell/540)

### CODE REVIEW

File: Google.PowerShell.IntegrationTests/BigQuery/BigQuery.BqJob.Tests.ps1 [hunk 1]
```diff
@@ -348,7 +348,7 @@ Describe "BqJob-Extract-Load" {
 
     AfterAll {
         $test_set | Remove-BqDataset -Force
-        $bucket.Name | Remove-GcsBucket -Force
+        Remove-GcsBucket "ps_test_$r" -Force
```
Comment (ILMTitan) [inline Google.PowerShell.IntegrationTests/BigQuery/BigQuery.BqJob.Tests.ps1]: This seems fragile and could break if someone changes the bucket name on the creation side. Why not use `$bucket.Name` or a new variable `$bucketName`?

File: Google.PowerShell.IntegrationTests/Container/Container.GkeCluster.Tests.ps1 [hunk 2]
```diff
@@ -246,27 +246,6 @@ Describe "Add-GkeCluster" {
...
-                             "subnetwork" = $subnetName;
-                             "additionalZone" = $clusterTwoAdditionalZone;
-                             "initialNodeCount" = 2;
-                             "machineType" = $clusterTwoMachineType }
-
-    $clusterTwoJob = Start-Job -ScriptBlock $clusterCreationWithoutUsingNodeConfigScriptBlock `
-                               -ArgumentList @($gcloudCmdletsPath, $clusterTwoParameter)
 
     # Cluster Three creation.
     $script:clusterThreeName = "gcp-new-gkecluster-3-$r"
```
Comment (ILMTitan) [inline Google.PowerShell.IntegrationTests/Container/Container.GkeCluster.Tests.ps1]: You should probably change some variable names so you still have sequential names (replace Three with Two).

File: Google.PowerShell/Compute/GceFirewallProtocolCmdlet.cs [hunk 3]
```diff
@@ -51,7 +62,7 @@ private class ParameterSetNames
         /// <para type="description">
         /// The ports which are allowed. This parameter is only applicable for UDP or TCP protocol.
         /// Each entry must be either an integer or a range. If not specified, connections through any port are
-        /// allowed Example inputs include: "22", "80","443", and "12345-12349".
+        /// allowed. Example inputs include: "22", "80","443", and "12345-12349".
```
Comment (ILMTitan) [inline Google.PowerShell/Compute/GceFirewallProtocolCmdlet.cs]: Add an example of the "1234..1239" notation.

**Comment by quoctruong** [comment]
PTAL

**Comment by ILMTitan** [comment]
LGTM.

**Comment by quoctruong** [comment]
Thanks!

**Pull request merged**